lstchain/image/muon/muon_analysis.py

```diff
@@ -291,3 +342,98 @@ def tag_pix_thr(image, thr_low = 50, thr_up = 500, pe_thr = 10):
     """
 
     return ((np.size(image[image > pe_thr]) < thr_up) and (np.size(image[image > pe_thr]) > thr_low))
+
+
+def radial_light_distribution(centre_x, centre_y, pixel_x, pixel_y, image):
+    """
+
+    Parameters
+    ----------
+    centre_x: float
+        Centre of muon ring in the field of view from circle fitting
+    centre_y: float
+        Centre of muon ring in the field of view from circle fitting
+    pixel_x: ndarray
+        X position of pixels in image
+    pixel_y: ndarray
+        Y position of pixel in image
+    image: ndarray
+        Amplitude of image pixels
+
+    Returns
+    -------
+    standard_dev, skewness
+    """
+    # Convert everything to degrees:
+    x0 = centre_x.to(u.deg).value
+    y0 = centre_y.to(u.deg).value
+    pix_x = pixel_x.to(u.deg).value
+    pix_y = pixel_y.to(u.deg).value
+
+    pix_r = np.hypot(pix_x-x0, pix_y-y0)
+
+    # mean, standard deviation & skewness of light distribution along ring radius.
+    # ring_radius calculated elsewhere is approximately equal to "mean", but not exactly, so we recalculate it here:
+    mean = np.sum(np.multiply(image,pix_r))/np.sum(image)
```

Comment:
use numpy average: `np.average(pix_r, weights=image)`, see e.g. the hillas parameter implementation in ctapipe:
https://github.com/cta-observatory/ctapipe/blob/master/ctapipe/image/hillas.py

Reply:
> use numpy average: `np.average(pix_r, weights=image)`, see e.g. the hillas parameter implementation in ctapipe:
> https://github.com/cta-observatory/ctapipe/blob/master/ctapipe/image/hillas.py

Ok, using numpy.average in all cases (use of cov does not seem clearer to me)